### PR TITLE
Restyle with `styler::style_pkg()`

### PR DESCRIPTION
**Pull request by olivroy**
Restyled the package with styler.

* Added a news bullet
* Replaced the `T`, `F` with `TRUE`, `FALSE`
* Whitespaces, indentation,  linebreaks
* Use `()` after pipes

See https://github.com/sfirke/janitor/pull/547#issuecomment-1584895857

In terms of automating the tidyverse style guide, there are 2 packages or actions that I found, but I am not very familiar with them so I would not feel comfortable implementing them in janitor.
* [lintr](lintr.r-lib.org)
* [precommit](https://lorenzwalthert.github.io/precommit/articles/precommit.html)

### CODE REVIEW

**Comment by billdenney** [comment]
@sfirke, any thoughts?

File: tests/testthat/test-tabyl.R [hunk 1]
```diff
@@ -296,7 +297,9 @@ test_that("NA levels get moved to the last column in the data.frame, are suppres
   expect_equal(names(y), c("down", "up", "NA_"))
   expect_equal(
     y[["NA_"]], # column c remains numeric
-    x %>% dplyr::filter(is.na(b)) %>% tabyl(c, a)
```
Comment (sfirke) [inline tests/testthat/test-tabyl.R:299]: It's interesting that it broke this pipeline up into multiple lines, there are others above that it left alone

File: tests/testthat/test-tabyl.R [hunk 2]
```diff
@@ -391,7 +395,7 @@ test_that("3-way tabyl with 3rd var factor is listed in right order, #250", {
   z$cyl[32] <- NA
   expect_equal(names(tabyl(z, am, gear, cyl)), c("4", "8", "6", "NA_"))
   expect_equal(names(tabyl(z, am, gear, cyl, show_na = FALSE)), c("4", "8", "6"))
-  z <- z %>% dplyr::filter(! cyl %in% "4")
+  z <- z %>% dplyr::filter(!cyl %in% "4")
```
Comment (sfirke) [inline tests/testthat/test-tabyl.R:398]: Like here's one of many pipelines on one line that it left alone

**Review by sfirke — APPROVED** [review]
Just flagged two things that I wanted to understand how they are consistent.  But it looks good to me and IMO it's not worth modifying to try to address my curiosity :)

**Comment by sfirke** [comment]
Agreed this is a nice boost to consistency.  I reviewed and in addition to what you noted Bill I saw a couple of equals-sign assignments replaced with `<-`.  Good stuff.  Let's merge it.

I do want to get something automated in place so that the repo can't get out of format and need a whopping big PR like this.  Thanks @olivroy for flagging precommit - that looks good to me.  The local install part seems a little odd, involving Python, but I gather we could set up [precommit lite](https://github.com/marketplace/actions/pre-commit-ci-lite) with a Github Action and it would make its own commits to PR branches that would run `styler` on the PR code.  I feel like that would be adequate for folks who don't want to mess with a local install of precommit?  The only part I don't understand from the docs is how to specify our action, i.e., run styler on all PRs.

**Comment by sfirke** [comment]
That can be a separate thing though, no need to hold up this PR.

**Comment by sfirke** [comment]
Hm some of these approaches to incorporating styler look reasonably easy: https://styler.r-lib.org/dev/articles/third-party-integrations.html  @olivroy are there any parameters you passed to styler or did you just use the default values?

**Comment by sfirke** [comment]
I added styler as a GitHub action using the command `usethis::use_github_action("style")`.  And then pushed directly to main which sorry,I should have made it a PR 😳   https://github.com/sfirke/janitor/commit/41f08c89419a5e55078d7fc42106de46559aa98e

File: tests/testthat/test-tabyl.R (hunk 2 above)
Comment (olivroy) [reply tests/testthat/test-tabyl.R:398]: according to the tidyverse style guide, 1 pipe is okay on a single line https://style.tidyverse.org/pipes.html#short-pipes

**Comment by olivroy** [comment]
Hi, I used the default parameters. 

I wouldn't mind readjusting in some places if you'd prefer

File: tests/testthat/test-tabyl.R (hunk 2 above)
Comment (sfirke) [reply tests/testthat/test-tabyl.R:398]: ah that was it! Thank you

**Pull request merged**